faculty_cli/cli.py

```diff
@@ -1158,7 +1165,7 @@ def put(project, local, remote, server):
                 "-P",
                 str(details.port),
                 os.path.expanduser(local),
-                u"{}@{}:{}".format(
+                "{}@{}:{}".format(
```

Comment:
We've not yet made the decision to drop Python 2 support. Can we leave these unicode literals until then?

Reply:
Looking back through the changes, I believe this is done by a newer version of black (than defined in `tox.ini`) cleaning that up, and when I realised that I run locally a different version and downgraded, I didn't go back to undo the changes that the original (version 19?) done. Totally makes sense, and will watch out for this later.